qa/tasks/cephfs/test_volumes.py

```diff
@@ -0,0 +1,306 @@
+import json
+import errno
+import random
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from teuthology.exceptions import CommandFailedError
+
+log = logging.getLogger(__name__)
+
+class TestVolumes(CephFSTestCase):
+    TEST_VOLUME_NAME = "fs_test_vol"
+    TEST_SUBVOLUME_PREFIX="subvolume"
+    TEST_GROUP_PREFIX="group"
+    TEST_SNAPSHOT_PREFIX="snapshot"
+
+    def _volume_cmd(self, *args):
+        return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
+
+    def _generate_random_subvolume_name(self):
+        return "{0}_{1}".format(TestVolumes.TEST_SUBVOLUME_PREFIX, random.randint(0, 10000))
+
+    def _generate_random_group_name(self):
+        return "{0}_{1}".format(TestVolumes.TEST_GROUP_PREFIX, random.randint(0, 100))
+
+    def _generate_random_snapshot_name(self):
+        return "{0}_{1}".format(TestVolumes.TEST_SNAPSHOT_PREFIX, random.randint(0, 100))
+
+    def _enable_multi_fs(self):
+        self._volume_cmd("flag", "set", "enable_multiple", "true", "--yes-i-really-mean-it")
```

Comment:
use `self.fs.set_var` instead

Reply:
I think we need to issue this via "ceph fs flag set" since `enable_multiple` is not supported via "ceph fs set" (used by `set_var`).

Reply:
Oh right, this is a different command. Sorry for the noise.